concept/type/relationtype.feature

```diff
@@ -213,6 +213,29 @@ Feature: Concept Relation Type and Role Type
     Then relation(parentship) get role(parent) is abstract: true
     Then relation(parentship) get role(child) is abstract: true
     Then relation(parentship) create new instance; throws exception
+    When session opens transaction of type: write
+    When relation(parentship) set abstract: false
+    When put relation type: fathership
+    When relation(fathership) set supertype: parentship
+    When relation(fathership) set relates role: father as parent
+    When relation(fathership) set relates role: father-child as child
+    When put entity type: person
+    When entity(person) set plays role: fathership:father
+    Then transaction commits
+    When connection close all sessions
+    When connection open data session for database: typedb
+    When session opens transaction of type: write
+    Then $m = relation(fathership) create new instance
+    When $a = entity(person) create new instance
+    When relation $m add player for role(father): $a
+    Then transaction commits
+    When connection close all sessions
+    When connection open schema session for database: typedb
+    When session opens transaction of type: write
+    Then relation(parentship) set abstract: true
+    Then transaction commits
+    When session opens transaction of type: read
+    Then relation(parentship) is abstract: true
```

Comment:
we confirm that we can freely change the parent to be abstract, since it still has no explicit instances